config/core/configmaps/autoscaler.yaml

```diff
@@ -130,3 +130,10 @@ data:
     # Scale to zero grace period is the time an inactive revision is left
     # running before it is scaled to zero (min: 30s).
     scale-to-zero-grace-period: "30s"
+
+    # Enable graceful scaledown feature flag.
+    # once enabled, it allows the autoscaler to priorities pods processing
+    # fewer (or no) requests over the others for removal, when scaling down.
+    # This improves convergence on the number of running pods and results in
+    # better resources utilization.
```

Comment:
```
    # Once enabled, it allows the autoscaler to prioritize pods processing
    # fewer (or zero) requests for removal when scaling down.
    # This improves convergence on the number of running pods and results in
    # better resources utilization.
```
I am not sure about "improves convergence" :-)

Reply:
got rid of the "why" part.